Classes/UT3Nightshade.uc

```diff
@@ -91,8 +92,39 @@ var float LastDropAttemptTime;
 var array< class<Actor> > MineObjectClasses;
 /* The mine that's on the arm when we deploy. */
 var Actor ArmMine;
+/* The third person camera distance before we deployed. */
+var float LastDesiredTPCamDistance;
+/* The third person camera distance to use when deployed. */
+var float DeployedTPCamDistance;
+/* The third person camera position to use when deployed. */
+var vector DeployedTPCamLookat;
 
 
+
+simulated function Destroyed() {
+	super.Destroyed();
+
+	if (ArmMine != none) {
+		ArmMine.Destroy();
+	}

```

Comment:
Braces for single lines are not needed.

Reply:
Another habit, will change! Think I may have some more that can be removed